client.go

```diff
@@ -325,48 +326,71 @@ func (c *Client) Close() error {
 
 // ConsumerFunc is passed into Consumer, and is used as a callback for incoming
 // Events. It should return true if the event was processed successfully and
-// false otherwise. If ConsumerUnsafe is being used the return is ignored
+// false otherwise.
 type ConsumerFunc func(Event) bool
```

Comment:
I was thinking about that, i wouldn't be opposed but it would break 3 or 4 of our projects until we fixed them